### PR TITLE
Helpful product reviews

### PR DESCRIPTION
**Pull request by maudnals**
This is an implementation of @andreban's use case idea. 

Input a product review, and get an on-device LLM-generated suggestion on how to make your review more helpful (or a thumbsup, if it was already helpful). This demo uses MediaPipe+Gemma1. Like our [other Gemma on-device demo](https://github.com/GoogleChromeLabs/web-ai-demos/tree/main/on-device-ai-perf-gemma), this demo offloads AI inference to a worker for better perf.

Some fixes still TBD, and the output is not 100% reliable (Gen AI). But this is a start. 
Refactoring/Cleanup still TODO in some places.

See README and demo screencast for the expected behavior and prerequisites.

<p align="center">
  
</p>

@andreban Possible follow-ups for this demo:
* Tweak this and add a server-side impl, as a hybrid AI proof-of-concept.
* Upgrade to Gemma 2 once available through MediaPipe LLM Inference API. My Gemma 1 prompt took a lot of iteration, but is still verbose and brittle.

### CODE REVIEW

File: helpful-product-reviews/src/index.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        <div>
+          <div id="modelStatusWrapper">
+            <div id="modelStatus"></div>
+          </div>
+          <div class="input-area">
+            <div>
+              <label for="reviewInputEl">Product review</label>
+            </div>
+            <textarea name="reviewInputEl" id="reviewInputEl">Too pricey</textarea>
+            <!-- <textarea name="test" id="test" oninput="handleUserInputChange()"></textarea> -->
```
Comment (tomayac) [inline helpful-product-reviews/src/index.html:30]:
```suggestion
```

File: helpful-product-reviews/src/index.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                <div id="reviewHelperStatus"></div>
+                <button id="inferenceButton" onclick="runLLMInference()">
+                  ▶️ Run review helper
+                </button>
+              </div>
+              <div id="reviewHelperOutput"></div>
+            </div>
+            <!-- <div>
+              <div id="llmOutput"></div>
+            </div> -->
```
Comment (tomayac) [inline helpful-product-reviews/src/index.html:47]:
```suggestion
```

File: helpful-product-reviews/src/script.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+function simulatePostReview() {
+  document.getElementById('reviewInputEl').value = '';
+  window.alert('Review posted!');
+}
+
+// Gen AI / inference + worker message handling
+
+function runLLMInference() {
+  const userPrompt = document.getElementById('reviewInputEl').value;
```
Comment (tomayac) [inline helpful-product-reviews/src/script.js:39]:
```suggestion
  const userPrompt = document.getElementById('reviewInputEl').value.trim();
```

**Review by tomayac — APPROVED** [review]
LGTM, modulu tiny nits

**Review by andreban — APPROVED** [review]
LGTM

**Pull request merged**